hebiros_advanced_examples/src/omnibase/Readme.md

```diff
@@ -0,0 +1,83 @@
+# Overview
+
+These files are used to interface with the omnibase robot base. This documentation provides instruction to run each of the nodes, as well as information on the ROS messages used by each file.
+
+**Important: to run any of these commands, you will need to run `source devel/setup.sh` from your catkin workspace first**
+
+
+# Initial Setup
+
+In a new directory (or your existing catkin workspace), run the following:
+
+```
+mkdir src
+cd src
+git clone https://github.com/HebiRobotics/HEBI-ROS.git
+git clone https://github.com/HebiRobotics/hebi_cpp_api_ros
+cd HEBI-ROS
+git checkout develop
+cd ..
+catkin_init_workspace
+cd ..
+```
+
+To build the catkin workspace, run the following commands:
+```
+catkin_make clean
+catkin_make
+source devel/setup.bash
+```
+
+# Running the code
+
+## Omnibase nodes:
+
+To run the omnibase odometry node using the keyboard teleop code, use this command:
```

Comment:
Ok - these tells me how to run each node, but not what they do.  Why is there a "free_roam" launch?  What does the "keyboard teleop" code do?  What is the "odometry node"?  How do I control the robot with the keyboard? etc.